4.1/4.1.9/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.1.9
+Создай функцию, которая будет возвращать объект с таким количеством пустых свойств, которое мы будем ей передавать.
+Все свойства должны быть названы и пронумерованы подобным образом: "prop-1", "prop-2" ...
+<script>
+	function getNewOnj(num) {
```

Comment:
`getObjWithEmptyProperties` would be better for the function naming
And think how to name the parameter `num` more better